base/BUILD.gn

```diff
@@ -1240,8 +1240,7 @@ component("base") {
 
   # Allow more direct string conversions on platforms with native utf8
   # strings
-  if (is_apple || is_chromeos || is_castos || is_cast_android || is_fuchsia ||
-      is_starboard) {
+  if (is_apple || is_chromeos || is_castos || is_cast_android || is_fuchsia) {
     defines += [ "SYSTEM_NATIVE_UTF8" ]
```

Comment:
What do we do today in trunk, do we define this or no?

Reply:
Nope, I'm following the trunk code in this PR.